commandapi-platforms/commandapi-bukkit/commandapi-bukkit-plugin/src/main/resources/config.yml

```diff
@@ -41,6 +41,13 @@ create-dispatcher-json: false
 # implementation is actually compatible with the current Minecraft version.
 use-latest-nms-version: false
 
+# Be lenient with version checks when loading for new minor Minecraft versions (default: false)
+# If "true", the CommandAPI loads NMS implementations for (potentially unsupported) Minecraft versions.
+# For example, this setting may allow updating from 1.21.1 to 1.21.2 as only the minor version is changing
+# but will not allow an update from 1.21.2 to 1.22
+# Keep in mind that implementations my vary and actually updating the CommandAPI might be necessary.
```

Comment:
Same note as `plugin-mojang-mapped`.

```suggestion
# but will not allow an update from 1.21.2 to 1.22.
# Keep in mind that implementations may vary and actually updating the CommandAPI might be necessary.
```

(_Sidenote outside of scope, but I wonder if there's an easy way for spigot and mojang-mapped plugin to share the config file resource, since they should be the same_)

Reply:
> (_Sidenote outside of scope, but I wonder if there's an easy way for spigot and mojang-mapped plugin to share the config file resource, since they should be the same_)

All we need is _yet another Maven module_ that they both inherit from 🫠

Reply:
I suppose one could perhaps inherit from the other. I think the only difference is the nms dependency, so as long as those get filtered correctly ¯\\\_(ツ)_/¯. That could also avoid duplicating CommandAPIMain and AdvancedConvertor.

Might also be possible to tell Maven to pull from the same shared resource folder.